app/helpers/admin/filter_helper.rb

```diff
@@ -18,14 +18,18 @@ def table_link_to(icon, text, path, options = {})
   private
 
   def filter_params(more_params)
-    params.permit(FILTERS).merge(more_params)
+    controller_request_params.permit(FILTERS).merge(more_params)
   end
 
   def filter_link_class(new_url)
-    filtered_url_for(params) == new_url ? 'selected' : ''
+    filtered_url_for(controller_request_params) == new_url ? 'selected' : ''
   end
 
-  def filtered_url_for(params)
-    url_for filter_params(params)
+  def controller_request_params
+    params.permit!
```

Comment:
This is a bad solution to this problem ... but it did get these specs passing.

This whole thing probably needs a rewrite, maybe separately and before this PR merged.